src/Servers/Kestrel/test/InMemory.FunctionalTests/Http2/Http2TestBase.cs

```diff
@@ -177,7 +178,11 @@ public Http2TestBase()
             _mockConnectionContext.Setup(c => c.Abort(It.IsAny<ConnectionAbortedException>())).Callback<ConnectionAbortedException>(ex =>
             {
                 // Emulate transport abort so the _connectionTask completes.
-                Task.Run(() => _pair.Application.Output.Complete(ex));
+                Task.Run(() =>
+                {
+                    TestApplicationErrorLogger.LogInformation($"ConnectionAbortedException thrown, Exception message: {ex.Message}. Completing transport Output.");
```

Comment:
This would omit the stack trace which could be useful. I also don't love how severe it looks considering it's pretty common for ConnectionContext.Abort() to be called. What about the following?

```suggestion
                TestApplicationErrorLogger.LogInformation(0, ex, "ConnectionContext.Abort() was called. Completing _pair.Application.Output.");
```

I haven't checked what the output looks like, but I think the test logs should include the full exception. If not, the other alternative is to use `{ex}` instead of `{ex.Message}` since the former just calls Exception.ToString().